client/components/marketing-survey/cancel-purchase-form/step-components/downgrade-step.jsx

```diff
@@ -50,7 +50,7 @@ export class DowngradeStep extends Component {
 			refundReason = (
 				<p>
 					{ translate(
-						'WordPress.com Personal still gives you access to email support, removal of ads, and more — and for 50% of the cost of your current plan.'
+						'WordPress.com Personal still gives you access to customer support via email, removal of ads, and more — and for 50% of the cost of your current plan.'
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.